docs/source/backends/raspbian.rst

```diff
@@ -0,0 +1,803 @@
+================
+Raspbian Backend
+================
+
+.. include:: ../_github.rst
+
+The ``Raspbian`` backend allows to Raspbian compatible configuration files.
+
+.. warning::
+    This backend is in experimental stage: it may have bugs and it will
+    receive backward incompatible updates in during the first 6 months
```

Comment:
argh, sorry, I suggested you a wrong description, it should be `it will receive backward incompatible updates during the first 6 months` (remove `in`)